src/src/AppContext.js

```diff
@@ -54,6 +55,7 @@ function AppProvider({ children }) {
 
   const [topics, setTopics] = useState([]);
   const [myCapabilities, setMyCapabilities] = useState([]);
+  const { falseUserPermissions } = useContext(PreAppContext);
```

Comment:
`falseUserPermissions` sounds somehow wrong?
Could we figure out a better naming -- maybe even consider an Enum